docfx_project/ziti/quickstarts/network/hosted.md

```diff
@@ -1,72 +1,73 @@
-the configuration options available you'll be better equipped to make changes. 
+When starting out deploying a [OpenZiti Network](xref:zitiOverview#overview-of-a-ziti-network), we recommend you follow
+and use the `expressInstall` function provided by the OpenZiti project. Once you're familiar with the network and
+the configuration options available you'll be better equipped to make changes.
 
 ### Firewall
 
-The first issue you will need to deal with is opening some ports. A network will consist of at least one controller and 
-at least one edge router. Both of these components will require ports to be open. For the controller you will need to 
-open two ports through your firewall, one port for the REST API, and one the control plane. Edge routers will also 
-require two ports open. One for the links created between routers to form the mesh network, and one port for incoming 
-client connections.
+The first issue you will need to deal with is opening some ports. The network's control plane is compose of a
+controller and at least one edge router. Both of these components will require server ports to be reachable by all
+components of your network. For the controller you will need to open two ports through your firewall, one port for
+the management REST API, and one the edge-client API. Edge routers will also require two ports open. One for the
+fabric links created between routers to form the mesh network, and one port for incoming edge-client connections.
 
 The ports you choose are not important but unless you change them these ports will default to the following:
 
 Controller
+
 - REST API: 1280 (port 8441 used below)
-- Control plane: 6262
+- Management API: 6262
 
 Edge Router
+
 - Edge connections: 3022 (port 8442 used below)
 - Fabric links: 10080
 
-It would be reasonable to change all these ports to use ports 443 and 80 since those ports are almost **always** open 
-outbound from all but the most strict networks. If you're going to be making connections from one of those networks, also 
+It would be reasonable to change all these ports to use ports 443 and 80 since those ports are almost **always** open
+outbound from all but the most strict networks. If you're going to be making connections from one of those networks, also
 verify that outbound access to these ports is available.
 
 ## Express Install
 
-With the firewall ports open you will now be able to source the script which provides the expressInstall function and 
-execute it. You probably should consider DNS names before actually running the script. By default, the script will use 
-the bash `hostname` function to determine a reasonable default for the PKI that will be generated as well as the 
-configuration files that are output.  The script allows you to configure these values and if you are deploying to a cloud 
-provider (AWS, Azure, OCI, IBM, Digital Ocean, GCP, etc.) you will almost certainly want to use the **public** DNS name 

```

Comment:
these linting changes are going to drive me crazy. please let's not be dogmatic about things like this?

Reply:
They would be tedious to undo at this point. Happy to talk more about when to lint to avoid unnecessary annoyances.

Reply:
right, my point is that I'd __really__ like to avoid this sort of change creeping in so beware of these sorts of changes. if you see your linter doing this sort of thing, it needs to change, or you can make formatting commit to "just add spaces to the ends of sentences" - which seems like a low-value PR to me :)